.github/workflows/apex-nightly.yml

```diff
@@ -16,8 +16,8 @@ jobs:
       e2e_polybft_test: true
       e2e_apex_test: true
       apex_bridge_ref: main
-      e2e_apex_big_test: ${{ vars.RUN_E2E_BIG_TESTS }}
-      e2e_apex_skip_redundant_tests: ${{ vars.SKIP_E2E_REDUNDANT_TESTS }}
+      e2e_apex_big_test: true
```

Comment:
also, maybe for starters dont run big tests, since they last longer than 6 hours - github will timeout